packages/toolpad-app/src/runtime/ToolpadApp.spec.tsx

```diff
@@ -155,7 +155,7 @@ test(`Databinding errors`, async () => {
       selfReferencing = appDom.createNode(dom, 'element', {
         name: 'selfReferencing',
         attributes: { component: 'Text' },
-        props: { value: { $$jsExpressione: 'selfReferencing.value' } },
```

Comment:
This typo wasn't failing the test before for some reason, but these changes made me find it.